src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -77,7 +77,7 @@
             az vm create -n MyVm -g MyResourceGroup --image UbuntuLTS
         - name: Create a default Windows Server VM with a private IP address.
           text: >
-            az vm create -n MyVm -g MyResourceGroup --public-ip-address "" --image Win2012R2Datacenter
+            az vm create -n MyVm -g MyResourceGroup --public-ip-address "" --image MicrosoftWindowsServer:WindowsServer:2012-Datacenter:latest
```

Comment:
Let us don't change this; rather create a new sample using URN with a different version to make it more real